### PR TITLE
modified the file, but the test failed.

### PR DESCRIPTION
**Pull request by Peng-Qu**
just a test to submit the code. not complete yet.

### CODE REVIEW

**Comment by JoshuaRM** [comment]
While reading through this @iblacker covered everything I had to say, and went the extra mile by including the individual code changes. Great work!

File: tests/spec/fs.chmod.spec.js [hunk 1]
```diff
@@ -70,4 +70,15 @@ describe('fs.chmod, fs.fchmod', function() {
       });
     });
   });
+
+  it('should err when path does not exist', function(done) {
+      var fs = util.fs();
+
+      fs.chmod('/', mode, function(error) {
```
[the remark this answers is not in the document]
Comment (Peng-Qu) [reply tests/spec/fs.chmod.spec.js]: Thank you so much.
I already fixed all and the test passed completely.

**Comment by Peng-Qu** [comment]
Thanks to  @iblacker for helping me with the codes.
Test now passed completely. 
BTW, do I need to open a new pull request?
Thanks.

**Comment by iblacker** [comment]
> Thanks to @iblacker for helping me with the codes.
> Test now passed completely.
> BTW, do I need to open a new pull request?
> Thanks.

That's great @Peng-Qu, I had the same question when I was working on mine. All you gotta do is commit the changes and push your branch to your origin. Then the Github will take care of the rest and update the pull request. Let me know if you have any questions

**Comment by humphd** [comment]
@Peng-Qu can you please commit and push your changes to this pull request so I can see the updated code?  All the review comments still stand until you submit new code.  Let us know if you need help doing this.

**Pull request closed without merging**